src/main/java/use_case/EventScheduler.java

```diff
@@ -17,29 +17,31 @@
     private ArrayList<Event> eventList;
```

Comment:
We can remove this attribute since we will not access the "database" directly from here anymore.

Reply:
Yep I can remove that. I think in the future we might rename this class to `CalendarAvailabilityAnalyzer` since that's the role it appears to be taking atm. I'll leave the name alone for this PR, though.